App/Sagas/TextileSagas.js

```diff
@@ -130,13 +130,18 @@ export function * photosTask (action) {
       yield call(RNFS.unlink, photo.thumbPath)
       photo['hash'] = multipartData.boundary
       yield put(TextileActions.imageAdded(photo, multipartData.payloadPath))
-      yield call(
-        UploadTask.uploadFile,
-        multipartData.payloadPath,
-        'https://ipfs.textile.io/api/v0/add?wrap-with-directory=true',
-        'POST',
-        multipartData.boundary
+      const uploadId = yield call(
+        Upload.startUpload,
+        {
+          path: multipartData.payloadPath,
+          url: 'https://ipfs.textile.io/api/v0/add?wrap-with-directory=true',
+          method: 'POST',
+          type: 'multipart',
+          field: multipartData.boundary
+        }
       )
+      photo['uploadId'] = uploadId
```

Comment:
This doesn't do anything, and you can't/shouldn't update any state in a saga anyways.

Reply:
ah, was replicating the photo['hash'] = something a few above. will drop that too